sdks/java/extensions/sketching/src/test/java/org/apache/beam/sdk/extensions/sketching/TDigestQuantilesTest.java

```diff
@@ -118,7 +118,7 @@ public void testMergeAccum() {
       accums.add(std);
     }
     TDigestQuantilesFn fn = TDigestQuantilesFn.create(100);
-    MergingDigest res = fn.mergeAccumulators(accums);
+    fn.mergeAccumulators(accums);
```

Comment:
Hm this actually looks like a bug. It should probably verify something about res (e.g. that it contains everything from accums). Could you try this:
```suggestion
    MergingDigest res = fn.mergeAccumulators(accums);
    assertThat(res.size(), equals(1000));
```

Reply:
Fixed